web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -160,6 +221,7 @@ import {
 import toggles from '@/featureToggle';
 
 import CopyText from '@/components/CopyText.vue';
+import _ from 'lodash';
```

Comment:
Can you check to see if you can extract just the functions you need to use? Something like

```typescript
import { isEmpty, map } from 'lodash';
```

I think this will help to keep the final bundle size small (but someone can correct me if I'm wrong).

Reply:
I was curious about this too, and it turns out there's [no difference](https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark) between the two here. Where there is a difference is when you do something like:
```typescript
import map from 'lodash/map';
import isEmpty from 'lodash/isEmpty';
```

There also may be ways to achieve the same effect with the `babel-plugin-lodash` and `lodash-webpack-plugin` modules. However, these are out of scope for the current issue.